modules/API/API.js

```diff
@@ -441,6 +441,27 @@ function initCommands() {
             });
             break;
         }
+        case 'get-livestream-url': {
+            const state = APP.store.getState();
+            const conference = getCurrentConference(state);
+            let livestreamUrl;
+
+            if (conference) {
+                const activeSession = getActiveSession(state, JitsiRecordingConstants.mode.STREAM);
+
+                if (activeSession && activeSession.id && activeSession.liveStreamViewURL) {
```

Comment:
You can simplify this by doing `activeSession?.liveStreamViewURL`

Reply:
Yes, I agree @saghul .
I can make that change as well, if you think that is required.
Jus let me know, Please.

Reply:
Yes please :-)

Reply:
Already, Done and pushed the changes below.